src/rpc/eth_api.rs

```diff
@@ -6,6 +6,11 @@ use crate::rpc_api::data_types::RPCState;
 use fvm_ipld_blockstore::Blockstore;
 use jsonrpc_v2::{Data, Error as JsonRpcError};
 
+// EthAccounts will always return [] since we don't expect Forest to manage private keys
```

Comment:
Should this be a `///`?

Reply:
Good question, I see both non-doc comments and doc comments for our RPC methods. In this case it's consistent with the Lotus codebase, but if you think this should be changed to a doc comment I'm happy to make the change.

Reply:
IMO if it's above the method signature, it should be a doc comment. If it's inside the method body, it should be an ordinary comment. I'm good with either approach. With the current phrasing, probably the latter.